environments/package.json

```diff
@@ -3,9 +3,9 @@
   "name": "environments",
   "version": "0.0.1",
   "scripts": {
-    "pull": "yarn auth pull && yarn geo pull && yarn datastore pull",
```

Comment:
Was it intentional to remove this script?

Reply:
Yes! It no longer works.